packages/graphback-codegen-schema/src/targetSchemaContext.ts

```diff
@@ -186,22 +215,22 @@ export const buildTargetContext = (input: InputModelTypeContext[]) => {
     return {
       "name": t.name,
       "type": 'input',
-      "fields": [...t.fields.filter((f: InputModelFieldContext) => f.type !== 'ID' && (!(f.isType && f.isArray) || !f.isType))
+      "fields": [...new Set([...t.fields.filter((f: InputModelFieldContext) => f.type !== 'ID' && (!(f.isType && f.isArray) || !f.isType))
```

Comment:
While this code works nicely this slowly becomes really hard to maintain. 
My main focus will be to get the cleaner path for relationships. I will share with you the idea once I will come up with solid plan